packages/plugin-compiler/src/main/kotlin/io/realm/compiler/RealmModelSyntheticPropertiesGeneration.kt

```diff
@@ -122,6 +123,8 @@ class RealmModelSyntheticPropertiesGeneration(private val pluginContext: IrPlugi
 
     private val listIrClass: IrClass =
         pluginContext.lookupClassOrThrow(FqNames.KOTLIN_COLLECTIONS_LIST)
+    private val kMutableProperty1Class: IrClass =
+        pluginContext.lookupClassOrThrow(FqNames.KOTLIN_REFLECT_KMUTABLEPROPERTY1)
     private val kProperty1Class: IrClass =
```

Comment:
Is this needed anymore if we now reference the mutable property instead?

Reply:
The mutable property was referenced before the PR, I introduced the non-mutable property reference.

Because the mutable property reference was named `kProperty1Class` I renamed it to `kMutableProperty1Class`, and the `kProperty1Class ` variable name is now used with the immutable.

Reply:
But I don't think this variable (`kProperty1Class`) is used anymore!?